Week1/homework/app.js

```diff
@@ -1,11 +1,149 @@
 'use strict';
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
-  ];
-
-  // Replace with your own code
-  console.log(bookTitles);
+//  1.1
+const bookTitles = [
+  'harry_potter_chamber_secrets',
+  'moll_flanders',
+  'joseph_andrews',
+  'jude_the_obscure',
+  'pride_and_prejudice',
+  'to_sir_with_love',
+  'to_miss_with_love',
+  'where_the_wild_things_are',
+  'me_talk_pretty_one_day',
+  'another_bullshit_night_in_suck_city'
+];
+
+//  1.2
+/*  function createBookList() {
```

Comment:
See my message in slack why this function should be present in your code (and not commented out).